__docs__/wonder-blocks-theming/theme-switcher-context.mdx

```diff
@@ -0,0 +1,48 @@
+import {Meta, Canvas} from "@storybook/blocks";
+
+import {
+    ThemeSwitcherContext,
+    createThemeContext,
+} from "@khanacademy/wonder-blocks-theming";
+
+import {ThemeSwitcherContextExample} from "./theme-examples";
+
+<Meta title="Theming / ThemeSwitcherContext" />
+
+# `ThemeSwitcherContext`
+
+`ThemeSwitcherContext` is a React Context that holds a reference to the selected
+theme. It should use default as the initial value as we expect that any
+theme-able component should define its initial theme as default.
```

Comment:
I think it might be more clear if this said \`default\` or "default" instead of plain default?